src/main/java/org/folio/circulation/domain/policy/library/ClosedLibraryStrategyService.java

```diff
@@ -120,10 +120,16 @@ private Result<ZonedDateTime> applyFixedDueDateLimit(
     ZonedDateTime dueDate, Loan loan, LoanPolicy loanPolicy, AdjacentOpeningDays openingDays,
     ZoneId timeZone, boolean isRecall){
 
+    ZonedDateTime loanOrDueDate;
```

Comment:
Maybe if we only have 3 cases - check out, recall, renewal - then we can do just if(!isRenewal && loanPolicy.isFixed) and not pass isRecall field any more?